bats/fb-content-katello.bats

```diff
@@ -121,8 +121,13 @@ EOF
     rpm -e `rpm -qf /etc/rhsm/ca/candlepin-local.pem`
   fi
 
+  yum erase -y 'katello-ca-consumer-*' || true
```

Comment:
There is a bats way to run a command and ignore its status code by using ```run```. I think that's cleaner than ```|| true```. Maybe the lines below should also be changed. It also makes me wonder when ignoring those status codes makes sense.